modules/scaling-cluster.adoc

```diff
@@ -6,23 +6,29 @@
 [id="scaling-cluster_{context}"]
 = Scaling your cluster
 
-You can scale your {product-title} cluster from the {OCM}.
+By using {OCM}, you can scale the number of load balancers, the persistent storage capacity, and the node count for your {product-title} cluster.
 
-.Procedure
+.Prerequisites
 
-. From {console-redhat-com}, click on the cluster you want to resize.
+* You logged in to {OCM}.
+* You created an {product-title} cluster.
 
-. Click *Actions* -> *Edit load balancers and persistent storage*
-.. Use the drop-down menu to select how many *Load balancers* you want to scale to.
-.. Use the drop-down menu to select the amount of *Persistent storage* you want to scale to.
-.. Click *Apply*. Scaling occurs automatically.
+.Procedure
 
-. Click *Actions* -> *Edit node count*.
-.. Use the drop down menu to select the *Machine Pool* setting you want to scale to.
-.. Use the drop down menu to select the *Node Count* you want to scale to.
-.. Click *Apply*. Scaling occurs automatically.
+* To scale the number of load balancers or the persistent storage capacity:
+. Navigate to {console-redhat-com} and select your cluster.
+. Select *Edit load balancers and persistent storage* from the *Actions* drop-down menu.
+. Select how many *Load balancers* that you want to scale to.
+. Select the *Persistent storage* capacity that you want to scale to.
+. Click *Apply*. Scaling occurs automatically.
 
+* To scale the node count:
+. Navigate to {console-redhat-com} and select your cluster.
+. Select *Edit node count* from the *Actions* drop-down menu.
+. Select a *Machine Pool*.
+. Select a *Node Count* per zone.
```

Comment:
It's `Node count` on UI rather than `Node Count`

Reply:
Thank you for highlighting this. Resolved.